openapi-module/src/GPT/config.js

```diff
@@ -0,0 +1,188 @@
+                    "type": "TEXT", // from end points above, generate random but realistic data for this
+                    "description": "" // from end points above, generate random but realistic data for this
+                  },
+                  {
+                    "key": "email", // from end points above, generate random but realistic data for this
+                    "value": "yujun150@gmail.com", // from end points above, generate random but realistic data for this
+                    "type": "TEXT", // from end points above, generate random but realistic data for this
+                    "description": "" // from end points above, generate random but realistic data for this
+                  }
+                ],
+                "header": [
+                  {
+                    "key": "Content-Type", // from end points above, generate random but realistic data for this
+                    "value": "application/json", // from end points above, generate random but realistic data for this
+                    "description": "" // from end points above, generate random but realistic data for this
+                  }
+                ],
+                "_id": "0bcecd91-12b7-48e9-b8d7-3b3fdd79af1d" // from end points above, generate random but realistic data for this
+              },
+              {
+                "scenarioId": "ba0323fb-a0f1-4c41-b573-cc92000a243f", // from end points above, generate random but realistic data for this
+                "requestName": "get", // from end points above, generate random but realistic data for this
+                "url": "http://www.example.com/", // from end points above, generate random but realistic data for this
+                "protocol": "http", // from end points above, generate random but realistic data for this
+                "host": "www.example.com", // from end points above, generate random but realistic data for this
+                "method": "GET", // from end points above, generate random but realistic data for this
+                "path": "/", // from end points above, generate random but realistic data for this
+                "port": "80", // from end points above, generate random but realistic data for this
+                "body": [], // from end points above, generate random but realistic data for this
+                "header": [], // from end points above, generate random but realistic data for this
+                "_id": "29f48f2b-3696-4834-a7a8-cbd46535f536" // from end points above, generate random but realistic data for this
+              }
+            ]
+          } and please remains the data integraty for generated json(aligh information)` ;
+      }
+  },
+
+  // output settings
+  outputDir: '../../temp',  // Directory to save the generated scenario
+  outputFileName: 'flex-scenario-gpt.flex',  // Default output file name with .flex extension
```

Comment:
The output filename could be set by an argument, eg. `--gptOutputFilename flex-scenario-gpt.flex`

Reply:
Update README file with samples of how to use this argument